tests/tools/nnpackage_run/src/args.cc

```diff
@@ -37,8 +37,10 @@ void Args::Initialize(void)
     ("help,h", "Print available options")
     ("version", "Print version and exit immediately")
     ("nnpackage", po::value<std::string>()->required())
+#if defined(ONERT_HAVE_HDF5) && ONERT_HAVE_HDF5 == 1
```

Comment:
(Optional) Should this macro have value 1? It seems defining macro w/o value would be enough.

Reply:
@periannath I like this style. Because I and someone like me expected `USE_NNAPI=0` would not use nnapi. However, it checked only whether `USE_NNAPI` is defined or not. Thus, it worked same to `USE_NNAPI=1`.